locales/en-US.json

```diff
@@ -28,7 +28,8 @@
     "muted_users": "Muted users",
     "muting": "Muted",
     "mutuals": "Mutuals",
-    "notify_on_post": "Notify me when {username} posts",
+    "notifications_on_post_disable": "Stop notifying me when {0} posts",
+    "notifications_on_post_enable": "Notify me when {0} posts",
```

Comment:
I do prefer going with the more explicit `{username}` interpolation but it does seem that list interpolation is the current pattern. I'll leave it up to @patak-dev to decide.

Reply:
{username} interpolation was used to make it clear what value is expected to be passed to the localization file.